### ISSUE TITLE
Contacts: No system message is displayed after removing a contact

### ISSUE THREAD

**Issue by glitchminer** [status-im/status-desktop#10498]
# Bug Report

Description
Split from and dependant on https://github.com/status-im/status-desktop/pull/10336

After removing a contact, a system notification is expected but not currently displayed

Steps to reproduce
Remove a contact via the contact's profile
Observe that no system message displayed

Additional Information
Status desktop version: https://ci.status.im/job/status-desktop/job/branches/job/macos/job/aarch64/job/package/job/PR-10336/20/
Operating System: Mac

**Comment by John-44** [comment]
@benjthayer does a user need to see a toast when *they* remove a contact *themselves*?   I was going to say "no we don't need a toast for this" (btw I think we definitely don't need a notification) but as we are using toasts to confirm user actions in many other places I wanted to ask for your thoughts on this quesion.

**Comment by benjthayer** [comment]
@MishkaRogachev happy to jump on a quick call to walk through these related flows which may be of use:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=27345-528266&t=whZ2rX5GbHFff1gY-4

**Comment by shinnok** [comment]
@glitchminer Hi,

I managed to get both the in-message-view notification and the toast for removing a contact, however, I'm assuming you opened this issue to also add an operating-system notification when a contact is removed?

PS: To clarify, code-wise, we refer to the _in-message-view_ notification as _system-message_.

<img width="843" alt="SCR-20231002-lcfq-2" src="https://github.com/status-im/status-desktop/assets/544446/391e910c-fa3d-496e-bd16-9d8303bcc875">

**Comment by jrainville** [comment]
> however, I'm assuming you opened this issue to also add an operating-system notification when a contact is removed?

I don't think so, since the rule for OS notifications to appear instead of Ephemeral/toast notifications is that the app in **not** in focus.

So in this case, the only way to remove a contact is using the app, then the app can only be in focus, so the Toast message is enough.

**Comment by shinnok** [comment]
In that case, we only need to add the missing toast notification, resolved in PR #12319.
